README.md

```diff
@@ -1 +1,6 @@
-# widgets
\ No newline at end of file
+# Widgets
+
+_Widgets_ is a UI software, providing  paramater input and control for the Pisces lightsheet microscope.
+
+The UI is started up by running widgets/PiscesInputParameterWidget.py. It depends on PyQt5 and Python 3.7, which can be installed using the requiremets.txt file in a Python 3.7 Anaconda environment. 
```

Comment:
Could you make it easier to read by putting in a format like in https://github.com/royerlab/dexp for example? Let me know if you have any markup questions.